.github/workflows/stack.yml

```diff
@@ -24,15 +24,18 @@ jobs:
           - resolver: lts-16
             ghc: 8.8.4
             yaml: stack-master.yaml
-          - resolver: lts-17
-            ghc: 8.10.4
-            yaml: stack-master.yaml
           - resolver: lts-18
             ghc: 8.10.7
             yaml: stack.yaml
           - resolver: lts-18
             ghc: 8.10.7
             yaml: stack-master.yaml
+          - resolver: lts-19
+            ghc: 9.0.2
+            yaml: stack-master.yaml
+          - resolver: lts-19
+            ghc: 9.0.2
+            yaml: stack-master.yaml
```

Comment:
Is this intentionally duplicated?

---

I should've added a comment back then about the stack/stack-master stuff: we primarily test using stack-master (uses X11, xmonad master) but we also want to make sure that it works with the latest X11, xmonad releases so we do that with the most and least recent LTS, just to make the test matrix a bit smaller (and thus less likely to hit github limits).

Reply:
Whoops, not at all!  Forgot to `stack-master.yaml -> stack.yaml` there